src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -0,0 +1,78 @@
+import Logger from "bunyan";
+import { BinaryLike, createHmac } from "crypto";
+import { Request, Response } from "express";
+import { validationResult } from "express-validator";
+import { getLogger } from "~/src/config/logger";
+import { GitHubServerApp } from "~/src/models/git-hub-server-app";
+import { WebhookContext } from "./webhook-context";
+
+export const WebhookReceiverPost = async (request: Request, response: Response): Promise<void> => {
+	const errors = validationResult(request);
+	if (!errors.isEmpty()) {
+		response.status(400).json({ errors: errors.array() });
+		return;
+	}
```

Comment:
we don't need this anymore as it's handled by the `returnOnValidationError` middleware

Reply:
I missed to remove it. Good catch